### PR TITLE
GH-15709: Craft Anova GLM Examples

### PR DESCRIPTION
**Pull request by shaunyogeshwaran**
#15709 
Crafted new examples to showcase the different parameters.

### CODE REVIEW

**Comment by wendycwong** [comment]
@shaunyogeshwaran : Thank you for working on this. However, I don't think we need an example for every parameters for ANOVAGLM. In fact, I think the only parameters you want to showcase are those specific to ANOVAGLM. What do you think?

**Comment by shaunyogeshwaran** [comment]
> What do you think?

I agree @wendycwong. I think crafting examples for each and every parameter makes the docs look tedious. I will go ahead and make the necessary changes to the other algorithms to only include what's unique to that algorithm.

**Review by hannah-tillman — DISMISSED** [review]

**Review by hannah-tillman — APPROVED** [review]

File: h2o-bindings/bin/custom/python/gen_anovaglm.py [hunk 1]
```diff
@@ -55,3 +56,103 @@ def result(self):
...
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    nparallelism=4)
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
+    prior="""
```
Comment (wendycwong) [inline h2o-bindings/bin/custom/python/gen_anovaglm.py:102]: Remove lines 88 to 101.  I will remove this parameter.

File: h2o-bindings/bin/custom/python/gen_anovaglm.py [hunk 2]
```diff
@@ -55,3 +56,103 @@ def result(self):
...
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    prior=0.0)
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()  
+""",
+    save_transformed_framekeys="""
```
Comment (wendycwong) [inline h2o-bindings/bin/custom/python/gen_anovaglm.py:116]: Remove lines 102 to 115.  The parameter prior is not specific to anovaglm.

File: h2o-bindings/bin/custom/python/gen_anovaglm.py [hunk 3]
```diff
@@ -55,3 +56,103 @@ def result(self):
...
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    save_transformed_framekeys=False)  
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
+    tweedie_link_power="""
+>>> import h2o
```
Comment (wendycwong) [inline h2o-bindings/bin/custom/python/gen_anovaglm.py:131]: remove lines 130 to 157, tweedie_link_power and tweedie_variance_power are not ANOVAGLM specific parameters.  They should be made into examples in GLM and not here.

File: h2o-bindings/bin/custom/python/gen_anovaglm.py [hunk 4]
```diff
@@ -55,3 +56,103 @@ def result(self):
...
+>>> train = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate_complete.csv.zip")
+>>> x = ['AGE', 'VOL', 'DCAPS']
+>>> y = 'CAPSULE'
+>>> anova_model = H2OANOVAGLMEstimator(family='binomial',
+...                                    lambda_=0,
+...                                    missing_values_handling="skip",
+...                                    save_transformed_framekeys=False)  
+>>> anova_model.train(x=x, y=y, training_frame=train)
+>>> anova_model.summary()
+""",
```
Comment (wendycwong) [inline h2o-bindings/bin/custom/python/gen_anovaglm.py:129]: change lines 117 to 128 with

>>> import h2o
 >>> h2o.init()
 >>> from h2o.estimators import H2OANOVAGLMEstimator 
 >>> train = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate_complete.csv.zip")
 >>> x = ['AGE', 'VOL', 'DCAPS']
 >>> y = 'CAPSULE'
 >>> anova_model = H2OANOVAGLMEstimator(family='binomial',
 ...                                    lambda_=0,
 ...                                    missing_values_handling="skip",
 ...                                    save_transformed_framekeys=False)  
 >>> anova_model.train(x=x, y=y, training_frame=train)
 >>>  transformFrame = h2o.get_frame(anova_model._model_json['output']['transformed_columns_key']['name'])
 >>> print(transformFrame)

**Review by wendycwong — CHANGES_REQUESTED** [review]

**Review by wendycwong — DISMISSED** [review]

**Review by hannah-tillman — APPROVED** [review]

**Pull request merged**